src/components/Form/Checkbox.js

```diff
@@ -1,30 +1,2 @@
-import React, { PureComponent } from "react";
-import PropTypes from "prop-types";
-import FormWrapper from "./FormWrapper";
-import CheckboxUI from "@material-ui/core/Checkbox";
-
-class Checkbox extends PureComponent {
-  static displayName = "Checkbox";
-
-  static defaultProps = {
-    checked: false
-  };
-
-  static propTypes = {
-    checked: PropTypes.bool,
-    handler: PropTypes.func.isRequired
-  };
-
-  handleOnChange = () => {
-    this.props.handler(!this.props.checked);
-  };
-
-  render() {
-    // eslint-disable-next-line
-    const { handler, ...props } = this.props;
-
-    return <CheckboxUI {...props} onChange={this.handleOnChange} />;
-  }
-}
-
-export default FormWrapper(Checkbox);
+import Checkbox from "@material-ui/core/Checkbox";
+export default Checkbox;
```

Comment:
Here migrated to Material UI. Removes the hassle of maintaining our own form components.